### DISCUSSION TITLE
Tests not populating https://openthread.io/testing

### DISCUSSION THREAD

**Discussion by ekobres** [openthread/openthread#8890] [General]
**Describe the bug** A clear and concise description of what the bug is.

https://openthread.io/testing has not been updated since July 2021 and none of the test data is populating the dashboards.

Is this set of dashboards still relevant?

**Comment by jwhui** [comment]
You are correct, we haven't generated new data in a while. It takes a non-trivial amount of resources to produce the data, maintain the test infrastructure, etc. Unfortunately, we haven't been able to resource that for some time.

OpenThread is a community-based open-source project. Any contributions are welcome!
